ci-operator/config/openshift/csi-operator/openshift-csi-operator-master.yaml

```diff
@@ -294,6 +306,22 @@ tests:
     env:
       TEST_SCENARIOS: \[CSI-Driver\]|\[Azure-File-CSI-Driver\]|\[Azure-File-CSI-Driver-Operator\]
     workflow: openshift-e2e-azure-csi-extended
+- as: e2e-openstack-cinder
```

Comment:
I know we're just porting whatever was in the cinder-csi-driver-operator repo, but What's this value of this job, if we have another one that runs CSI specific tests?

Reply:
Haven't a clue, though I do note that we currently do similar things for AWS, Azure, etc.

Maybe a question for @gnufied and @jsafrane?

Reply:
How is this job different from `e2e-openstack-cinder-csi` ?

Reply:
I renamed the job but kept it for now. Let's determine whether we want to keep it long-term post-merge?

Reply:
All components that are installed by default _should_ run regular e2e test to make sure that a CSI driver update does not break other functionality of the cluster, dunno... Routes or Networking. The probability is IMO very small here.

And the same for upgrades - all components that are installed by default _should_ run regular e2e-upgrade job to ensure a CSI driver change does not break upgrade. That's much more likely to happen, we had cases when a driver upgrade broke an existing cluster.

These rules were documented somewhere, but I can't find it now :-(